www/changelog.html

```diff
@@ -92,6 +92,22 @@ <h3>Support managing source component on the GUI</h3>
       </div>
     </div>
 
+    <div class="intro-row">
+      <h3>Analysis Statistics</h3>
+      <div class="content">
+        <div class="intro-col intro-col-left col-md-2">
+          It is displayed for each run, which CodeChecker and Clang Analyzer
```

Comment:
I would rephrase the short description a bit:
`For each run the CodeChecker version (used to run the analysis) and the static analyzer versions along with the success/failure rate of the analysis can be viewed.`